personal_graph/natural.py

```diff
@@ -16,19 +16,30 @@
 
 load_dotenv()
 
-if os.getenv("OPEN_API_KEY"):
-    client = instructor.patch(OpenAI(api_key=os.getenv("OPEN_API_KEY")))
+if os.getenv("LITE_LLM_BASE_URL"):
```

Comment:
strange choice, ideally there should be no choices made during import time. Currently this code gets run at import time while, one would want to pick it when constructing a graph